admin_manual/configuration_server/server_tuning.rst

```diff
@@ -73,6 +99,27 @@ Enable HTTP2 for faster loading
 
 HTTP2 has `huge speed improvements <https://www.troyhunt.com/i-wanna-go-fast-https-massive-speed-advantage/>`_ over HTTP with multiple request. Most `browsers already support HTTP2 over SSL (HTTPS) <http://caniuse.com/#feat=http2>`_. So refer to your server manual for guides on how to use HTTP2.
 
+Tune PHP-FPM
+------------
+
+If you are using a default installation of php-fpm you might have noticed 
+excessive load times on the web interface or even sync issues. This is due 
+to the fact that each simultaneous request of an element is handled by a 
+separate PHP-FPM process. So even on a small installation you should allow 
+more processes to run. For example on a machine with 4GB of RAM and 1GB of 
+MySQL cache following values in your ``www.conf`` file should work:
+
+.. code:: ini
+
+  pm = dynamic
+  pm.max_children = 120
+  pm.start_servers = 12
+  pm.min_spare_servers = 6
+  pm.max_spare_servers = 18
+  pm.process_idle_timeout = 60s
```

Comment:
> Used only when pm is set to ondemand.

(from http://php.net/manual/en/install.fpm.configuration.php)

Reply:
ah, right pm.process_idle_timeout is unnecessary